routes/dao_coin_exchange_with_fees.go

```diff
@@ -524,10 +534,10 @@ func (fes *APIServer) GetQuoteCurrencyPriceInUsd(
 
 	// If the profile is the dusdc profile then just return 1.0
 	lowerUsername := strings.ToLower(string(existingProfileEntry.Username))
-	if strings.Contains(lowerUsername, "dusdc") {
+	if lowerUsername == "dusdc_" {
```

Comment:
I like this much better! we just need to remember to adjust it if we ever change the profile name.